addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoBridgeHandler.java

```diff
@@ -65,6 +65,7 @@ public void initialize() {
             getPartnerApi().partnerdevices();
         } catch (RetrofitError e) {
             if (e.getCause() instanceof IOException) {
+                logger.error("Unable to connect Netatmo API : {}", e.getMessage(), e);
```

Comment:
The error is already reported to the thing status, can you lower the log level, especially given:

In [E.4 of the coding guidelines](http://docs.openhab.org/developers/development/guidelines.html#e-logging) we listed what kind of logging level should be used when. Error should only be used:

> `error` logging should only be used to inform the user that something is tremendously wrong in his setup, the system cannot function normally anymore, and there is a need for immediate action. 

Please change this to use the correct logging levels and if possible change the ONLINE/OFFLINE status of thing / bridge.